updater/run.sh

```
@@ -98,6 +99,45 @@ drop_to_project_owner() {
+      if [ "${tracked_uid}" != "${project_uid}" ]; then
+        rm -f -- "${tracked_list}"
+        echo "ERR: checkout owner UID ${project_uid} differs from tracked path owner UID ${tracked_uid:-unknown} at ${tracked_path}; use host CLI recovery" >&2
+        return 2
+      fi
+    fi
+    tracked_parent="$(dirname "${tracked_path}")"
+    while [ "${tracked_parent}" != "${PROJECT_ROOT}" ]; do
+      if [ -e "${tracked_parent}" ] || [ -L "${tracked_parent}" ]; then
+        tracked_uid="$(stat -c '%u' "${tracked_parent}" 2>/dev/null || true)"
+        if [ "${tracked_uid}" != "${project_uid}" ]; then
+          rm -f -- "${tracked_list}"
```

Shellcheck (0.11.0)

[info] Make sure not to read and write the same file in the same pipeline.

(SC2094)